iModelCore/Units/tests/NonPublished/FormattingTests.cpp

```diff
@@ -1738,4 +1738,87 @@ TEST_F(FormattingTestFixture, AzimuthWithVariousBases) {
     ASSERT_STREQ("180.0°", formatAzimuth(90.0, 270.0,true).c_str());
 }
 
+TEST_F(FormattingTestFixture, FormatDoubleToFractionalRatio){
+    auto formatFractionalRatio = [](double value, Units::UnitCP unit) -> Utf8String
+    {
+        NumericFormatSpec fractionalRatioSpec;
+        fractionalRatioSpec.SetPresentationType(PresentationType::FractionalRatio);
+        fractionalRatioSpec.SetPrecision(DecimalPrecision::Precision3);
+
+        Format fractionalRatioFormat(fractionalRatioSpec);
+
+        Units::Quantity quantity(value, *unit);
+        return fractionalRatioFormat.FormatQuantity(quantity);
+    };
+
+    // Vertical per Horizontal
+    {
+        auto verticalPerHorizontalUnit = s_unitsContext->LookupUnit("VERTICAL_PER_HORIZONTAL");
+
+        ASSERT_STREQ("0:1", formatFractionalRatio(0.0, verticalPerHorizontalUnit).c_str());

```

Comment:
The way Simon described it on the main story, @ColinKerr derived the following conclusion: https://github.com/iTwin/itwinjs-backlog/issues/1147#issuecomment-2217975255
So for integer ratio, you should not always have a "1" on one side of the ratio.
It's confusing, researching the web I find different definitions for integer/fractional ratio. Can you start a discussion and make sure we have some good test data to work with? We need to better understand the expected solution.